src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,125 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final long DEFAULT_ID = 100L;
+    private static final String DEFAULT_LOGIN = "login373";
+    private static final String SHORT_LOGIN = "login";
+    private static final String DEFAULT_PASSWORD = "password373";
+    private static final String SHORT_PASSWORD = "pass";
+    private static final int DEFAULT_AGE = 20;
+    private static final Integer MINIMAL_AGE = 18;
+    private static final Integer INVALID_AGE = 15;
```

Comment:
I suppose it'd be better if you add tests for boundary cases 
For exp : 5 symbols password and 6 symbols password to check that validation works correctly
And for login the same